editor/editor_node.h

```diff
@@ -140,6 +140,17 @@ class EditorNode : public Node {
 		SCENE_NAME_CASING_KEBAB_CASE,
 	};
 
+	enum ActionOnPlay {
+		DO_NOTHING = 0,
+		OPEN_OUTPUT = 1,
+		OPEN_DEBUGGER = 2,
+	};
+
+	enum ActionOnStop {
+		DO_NOTHING = 0,
```

Comment:
Due to how enums work in C++ you should either prefix them with name (e.g. `STOP_DO_NOTHING`), or I think using `enum class` would fix it too.

Reply:
Yeah, I'll adjust, I'm not very experienced with different versions of C++, it was working for me, I'll adjust.